crates/router/src/services/api.rs

```diff
@@ -1824,13 +1843,12 @@ pub fn build_redirection_form(
                     const threeDSsecureInterface = threeDS.createUI(options);
 
                     threeDSsecureInterface.on('challenge', function(e) {{
-                        document.getElementById('loader-wrapper').style.display = 'none';
```

Comment:
Can we revert this change?

Reply:
This change is not related to my PR somehow this change is showing up, though this portion of my file is in Sync with main. Have tested NMI where this change lies also verified and took approval from @Sakilmostak .

Reply:
Ideally, this shouldn't be showing up in the diff at all. Can we just pick the entire block/function from the `main` branch using the GitHub web UI and include it in your branch, if that helps?

Reply:
i did copy paste the same UI changes from the main branch still i can see the same.